openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -27,10 +27,6 @@
 // Developer styles for Staff Section
 // --------------------
 
-// edX constants
-$lighter-base-font-color: rgb(100,100,100);
-$link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.org/elements/colors/
-
 .wrapper--openassessment .wrapper--staff-area {
```

Comment:
Nit: at some point we should remove this file since we don't have a UX/developer distinction anymore for Sass development. It might be a good time to make this change now, but I understand if that seems out-of-scope.